src-internal/Admin/WcPayPromotion/WcPaymentGatewayPreInstallWCPayPromotion.php

```diff
@@ -16,7 +16,7 @@
  *
  * @extends WC_Payment_Gateway
  */
-class WCPaymentGatewayPreInstallWCPayPromotion extends \WC_Payment_Gateway {
+class WcPaymentGatewayPreInstallWCPayPromotion extends \WC_Payment_Gateway {
```

Comment:
I think it would be great to update the class name in the comments [here](https://github.com/woocommerce/woocommerce-admin/blob/d52410a9c9a3bbc51337df2ff618a893606062a5/src-internal/Admin/WcPayPromotion/WcPaymentGatewayPreInstallWCPayPromotion.php#L3).

Reply:
I'm also curious why do we use `Wc` prefix but have **WC**PayPromotion and **WC**_Payment_Gateway?

Reply:
Thank you @chihsuan 

That's a good question. I think it's something we need to discuss and reach an agreement. I don't have a strong preference.

The only reason I renamed it to `Wc` is that the directory and the other class have `Wc` already. It's mixed at the moment.

Let's change it to `WC` since we're moving. I'll make an update.

Reply:
@chihsuan Updated letter case in https://github.com/woocommerce/woocommerce-admin/pull/8307/commits/a03f05d0255b94d66e0c458052a5e26c74f4d4ab and https://github.com/woocommerce/woocommerce-admin/pull/8307/commits/5a20f6278f8e5c7fdc9fb4e00269a5318a3860f3

I also tested the changes on a JN site since my local mac's filesystem isn't case sensitive.

Reply:
FYI, I'm gonna wait until your PRs are ready to merge since those PRs are merging into this PR.

Reply:
> Updated letter case in https://github.com/woocommerce/woocommerce-admin/commit/a03f05d0255b94d66e0c458052a5e26c74f4d4ab and https://github.com/woocommerce/woocommerce-admin/commit/5a20f6278f8e5c7fdc9fb4e00269a5318a3860f3

> I also tested the changes on a JN site since my local mac's filesystem isn't case sensitive.

@moon0326 Thanks for updating letter case. Looks good. 💯

Reply:
@moon0326 I think you can merge this PR first. I'll change the merge target for the other PRs.

Reply:
@chihsuan sounds good. Merged!